locations/spiders/cafe_zupas.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+import scrapy
+from locations.items import GeojsonPointItem
+
+
+class CafeZupasSpider(scrapy.Spider):
+    name = "cafe_zupas"
+    item_attributes = {"brand": "Cafe Zupas"}
+    allowed_domains = ["cafezupas.com"]
+    start_urls = [
+        "https://cafezupas.com/server.php?url=https://api.controlcenter.zupas.com/api/markets/listing"
+    ]
+
+    def parse(self, response):
+        data = response.json()
+        for i in data["data"]["data"]:
+            for location in i["locations"]:
+                properties = {
+                    "ref": "https://cafezupas.com/locationcopy/info/"
+                    + location["name"].lower().replace(" ", "-"),
+                    "website": "https://cafezupas.com/",
```

Comment:
It looks like each location has its own website and you have it going to `ref` right now. Could you move that here?